apps/storybook/src/feedback/Toast.stories.tsx

```diff
@@ -81,10 +82,10 @@ export const Appearance: Story = {
         </Button>
         <Button
           onClick={() =>
-            toaster.create(<Toast {...args} colorScheme="success" />)
+            toaster.create(<Toast {...args} colorScheme="informational" />)
```

Comment:
this should be `information`
```suggestion
            toaster.create(<Toast {...args} colorScheme="information" />)
```